core/src/mast/node/call_node.rs

```diff
@@ -58,24 +69,48 @@ impl CallNode {
     }
 }
 
+//-------------------------------------------------------------------------------------------------
+/// Public accessors
 impl CallNode {
+    /// Returns a commitment to this Call node.
+    ///
+    /// The commitment is computed as a hash of the callee and an empty word ([ZERO; 4]) in the
+    /// domain defined by either [Self::CALL_DOMAIN] or [Self::SYSCALL_DOMAIN], depending on
+    /// whether the node represents a simple call or a syscall - i.e.,:
+    ///
+    /// hasher::merge_in_domain(&[callee_digest, Digest::default()], CallNode::CALL_DOMAIN)
+    ///
+    /// or
+    ///
+    /// hasher::merge_in_domain(&[callee_digest, Digest::default()], CallNode::SYSCALL_DOMAIN)
```

Comment:
Should we make these proper Markdown code blocks so that `cargo doc --test` catches when the API changes?

```suggestion
    /// ```rs
    /// hasher::merge_in_domain(&[callee_digest, Digest::default()], CallNode::CALL_DOMAIN)
    /// ```
    ///
    /// or
    /// 
    /// ```rs
    /// hasher::merge_in_domain(&[callee_digest, Digest::default()], CallNode::SYSCALL_DOMAIN)
    /// ```
```

Reply:
Same comment applies to all other node types

Reply:
Good idea! Converted!